Cartfile.resolved

```diff
@@ -1,6 +1,6 @@
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json" "6.32.0"
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashlyticsBinary.json" "6.32.0"
-github "Alamofire/Alamofire" "5.2.2"
+github "Alamofire/Alamofire" "5.4.0"
```

Comment:
@justinswart indeed, I've been considering pushing this, but I wasn't sure if this bump doesn't break anything 👌.

Reply:
I think this is fine, we only use AlamofireImage for some simple image caching - so this is included by way of that dependency.